app/index.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app");
+const port = 3000;
+
+app.listen(port, () => {
```

Comment:
What if there will be an error while starting the server?

Reply:
add handling error